feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -203,7 +203,7 @@ fun ToggleQuickSettingsButton(toggleDropDown: () -> Unit, isOpen: Boolean) {
             painter = painterResource(R.drawable.baseline_expand_more_72),
             contentDescription = stringResource(R.string.quick_settings_dropdown_description),
             modifier = Modifier
-                .testTag("QuickSettingDropDown")
+                .testTag("QuickSettingsDropDown")
```

Comment:
Use the constants here rather than string literals.

Reply:
#133 adds a `TestTags.kt` for the `feature:quicksettings` module. We should move the test tags there and then they can be referenced more easily.

Reply:
Just to be clear- only the quick settings test tags should be moved there. We should have TestTags in each module for the components that exist in that module.

Reply:
This still needs to be changed to `QUICK_SETTINGS_DROP_DOWN`